:memo: Update firewall docs (#2204)
platformsh/platformsh-docs: docs/src/configuration/app/app-reference.md

| --------- | ------------------- | --------------- | ----------- |
| `ips`     | Array of `string`s  | `["0.0.0.0/0"]` | IP addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). See a [CIDR format converter](https://ipaddressguide.com/cidr). |
| `domains` | Array of `string`s  |                 | [Fully qualified domain names](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) to specify specific destinations by hostname. |

> | `ports`   | Array of `integer`s |                 | Ports from 1 to 65535 that are allowed. If specified, unspecified ports are blocked. If no ports are specified, all ports are allowed. Port `25`, the SMTP port for sending email, is always blocked. |

Reviewer: ```suggestion
| `ports`   | Array of `integer`s |                 | Ports from 1 to 65535 that are allowed. If ports are specified, all unspecified ports are blocked. If no ports are specified, all ports are allowed. Port `25`, the SMTP port for sending email, is always blocked. |
```

Author: It would more technically be "If this property is specified" and I don't think that adds a lot of clarity, so I'd prefer the shorter one, unless you can think of some ambiguity that people might actually take from it?

Reviewer: It took me a few times to re-read the sentence to be able to understand its meaning. 

```suggestion
| `ports`   | Array of `integer`s |                 | If specified, unspecified ports are blocked. If no ports are specified, all ports are allowed. By default, ports from 1 to 65535 are allowed with the exception of the always blocked port `25` (SMTP port for sending email). |
```

Author: This way is more confusing because it puts what can be specified there after it describes the consequences. Leading with what can be there seems better to me. I feel like the original is common phrasing in docs in general, including ours. Like here: https://github.com/platformsh/platformsh-docs/blob/main/contributing/markup-format.md?plain=1#L274

But I can change it.

Reviewer: okey yeah, that makes sense 🤔 
Thanks for adding the additional words in the latest push, it seems clearer at the first reading occurrence of that sentence which might help non-native speakers.

so thousands of domain names may share the same public IP addresses at the CDN.
If you allow the IP address of a CDN, you are usually allowing many or all of the other customers hosted behind that CDN.

### Outbound traffic by domain

You can filter outbound traffic by domain.
Using domains in your rules rather than IP addresses is generally more specific and secure.
For example, if you use an IP address for a service with a CDN,
you have to allow the IP address for the CDN.

> THis means that you allowing potentially hundreds or thousands of servers also using the CDN.

Author: I'd keep the adverb after "allow" because it's definitely allowing them, it's just a question of number.
